website/docs/components/dropdown/partials/code/component-api.md

```diff
@@ -104,10 +104,13 @@ If the Dropdown content exceeds the height of the container, the header and foot
     Text to be used in the item. If no text value is defined, an error will be thrown.
   </C.Property>
   <C.Property @name="color" @type="enum" @values={{array "action" "critical" }} @default="action">
-    Acceptable values: “action” or “critical”.
+    Color applied to the text and (optional) icons. Acceptable values: “action” or “critical”.
```

Comment:
_Acceptable values: “action” or “critical”._ feels redundant now that we show values as part of the ComponentApi Properties – as we do for other arguments
<img width="748" alt="Screenshot 2024-04-09 at 10 37 31" src="https://github.com/hashicorp/design-system/assets/788096/ff103d1d-78ca-4ba8-b322-b6da282633ba">

Reply:
Done.